components/docs-chef-io/content/automate/data_lifecycle.md

```diff
@@ -43,6 +43,29 @@ The default is to remove Chef Infra Client run data after 30 days, to label node
 The Compliance data lifecycle settings allow you to remove compliance reports and compliance scans after a set amount of days.
 The default is to remove compliance reports after 60 days, and to remove compliance scans after 60 days.
 
+## Max Shards
+
+The max shards are the number of shards that should be patched to run the data lifecycle. If the client runs and the compliance data lifecycle is set to 30 days, the max shards per node configuration should be 1000. Refer to the following table for calculated max shards for the client run and compliance data lifecycle.
```

Comment:
The statement seems like that the patching is necessary before setting the life cycle. That is not the case. The numbers should be adjusted in order to avoid overloading of shards. @YashviJain01  Please share the error the user gets in case of overloaded shards. We can document it in such manner.